cfgmgr/vrfmgr.cpp

```diff
@@ -0,0 +1,146 @@
+#include <string.h>
+#include "logger.h"
+#include "dbconnector.h"
+#include "producerstatetable.h"
+#include "tokenize.h"
+#include "ipprefix.h"
+#include "vrfmgr.h"
+#include "exec.h"
+#include "shellcmd.h"
+
+#define VRF_TABLE_START 100
+#define VRF_TABLE_END 164
```

Comment:
what are these two numbers? does this limit to 64 vrfs?

Reply:
Linux routing table numbers. Do we need more?

Reply:
maybe can change to 1k initially.

Reply:
Done
Used 1001..2000 since there are a few reserved values in the first 256 tables.